src/effects/VST/VSTEffect.h

```diff
@@ -100,7 +100,14 @@ struct VSTEffectSettings
 };
 
 
-struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
+struct VSTEffectUIWrapper
+{
+   virtual void NeedIdle();
+   virtual void SizeWindow(int w, int h);
```

Comment:
I think this interface is insufficient.  I think `NeedEditIdle()` and `Automate()` must be part of it too.

Reply:
now done